src/components/SingleTrikot.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import {List, Image,Container} from "semantic-ui-react";
+import store from '../store'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'store'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)